front-end/studio/src/app/pages/apis/{apiId}/_components/generate-project.wizard.ts

```diff
@@ -61,7 +61,8 @@ export class GenerateProjectWizardComponent {
     @Input() apiId: string;
 
     protected _isOpen: boolean = false;
-
+   
+    public validatePattern = new RegExp(/((http|git|ssh|http(s)|file|\/?)|(git@[\w\.]+))(:(\/\/)?)([\w\.@\:/\-~]+)(\.git)(\/)?/);
```

Comment:
should be https link only, please remove http, git, ssh file, etc, from this Regex.
use, for example https://regex101.com/ to test your expression 
@AndreiZiatiev

Reply:
done